setup.py

```diff
@@ -904,6 +894,7 @@ def filter_files(path_obj, all_files, allowed_files, disallowed_files):
             commands = {
                 "clang-format": ["--dry-run", "--Werror", "-i"] + c_files,
                 "black": ["--check", "--diff"] + python_directories,
+                "codespell": [],  # Codespell configuration is in setup.cfg 
```

Comment:
I would suggest that the skip files are passed here via the command line.

The advantage of doing this is that we can re-use the `c_file_disallow` already defined here